package.json

```diff
@@ -66,23 +67,23 @@
     "@nrwl/nx-cloud": "^15.1.0",
     "@nrwl/nx-plugin": "14.8.8",
     "@nrwl/react": "14.8.8",
-    "@nrwl/storybook": "14.8.8",
+    "@nrwl/storybook": "15.9.2",
     "@nrwl/web": "14.8.8",
     "@nrwl/workspace": "14.8.8",
     "@nxext/stencil": "14.0.6",
     "@stencil/angular-output-target": "0.6.0",
     "@stencil/core": "3.2.1",
     "@stencil/react-output-target": "0.5.0",
     "@stencil/sass": "3.0.1",
-    "@storybook/addon-a11y": "6.5.16",
-    "@storybook/addon-actions": "6.5.16",
-    "@storybook/addon-docs": "6.5.16",
-    "@storybook/addon-essentials": "6.5.16",
-    "@storybook/addon-links": "6.5.16",
-    "@storybook/builder-webpack5": "6.5.16",
-    "@storybook/manager-webpack5": "6.5.16",
+    "@storybook/addon-a11y": "7.0.5",
+    "@storybook/addon-actions": "7.0.5",
+    "@storybook/addon-docs": "7.0.5",
+    "@storybook/addon-essentials": "7.0.5",
+    "@storybook/addon-links": "7.0.5",
+    "@storybook/addon-mdx-gfm": "^7.0.5",
```

Comment:
```suggestion
    "@storybook/addon-mdx-gfm": "7.0.5",
```

Reply:
Solved. The dependency was removed, and no longer needed.